shared/js/https.js

```diff
@@ -1,228 +1,211 @@
 (function () {
-const db = require('db')
+const load = require('load')
 const settings = require('settings')
 const utils = require('utils')
-let knownMixedContentList
 
-settings.ready().then(() => {
-    load.JSONfromLocalFile(constants.httpsWhitelist, (wl) => knownMixedContentList = wl)
-})
+// number of chunks to split up the data
+// when storing to Safari to avoid over quota issues:
```

Comment:
Technical doc link for future reference?

Reply:
will add it